modules/launcher/src/main/java/org/mule/module/launcher/artifact/AbstractArtifactClassLoader.java

```diff
@@ -24,12 +27,18 @@
  */
 public abstract class AbstractArtifactClassLoader extends FineGrainedControlClassLoader implements ArtifactClassLoader
 {
+    private static final String DEFAULT_LEAK_CLEANER_CLASS = "DefaultLeakCleaner.class";
+
     protected Log logger = LogFactory.getLog(getClass());
 
     protected List<ShutdownListener> shutdownListeners = new ArrayList<ShutdownListener>();
 
     private LocalResourceLocator localResourceLocator;
 
+    private LeakCleaner leakCleaner;
```

Comment:
I don't see any reason why there should be a LeakCleaner field because that instance is used only during classlaoder disponsig